src/main/java/fileservice/CsvFileService.java

```diff
@@ -0,0 +1,7 @@
+package fileservice;
+
+import java.util.List;
+
+public interface CsvFileService {
```

Comment:
```suggestion
public interface ReadFileService {
```

there can be many implementations of this interface. and file extension should be in implementation class name